addons/dev-helpers/bin/mk-pfmon-task.pl

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/perl
```

Comment:
The script will be removed when the old values of
is removed conf/pf.conf.defaults conf/documentation.conf